lnbits/wallets/fake.py

```diff
@@ -45,19 +45,18 @@ async def create_invoice(
     ) -> InvoiceResponse:
         data: Dict = {
             "out": False,
-            "amount": amount,
+            "amount": amount * 1000,
```

Comment:
- this looks like a major bug that we should have noticed for the `FakeWallet`
- how did it manifest before this change?